src/metabase/driver/crate/native.clj

```diff
@@ -0,0 +1,45 @@
+(ns metabase.driver.crate.native
+  (:require [clojure.java.jdbc :as jdbc]
+            [metabase.driver :as driver]
+            [metabase.models.database :refer [Database]]
+            [metabase.db :refer [sel]]
+            [metabase.driver.generic-sql :as sql]
+            [clojure.tools.logging :as log]
+            [metabase.util :as u]))
+
+(defn- value->base-type
```

Comment:
this function definition is duplicate in our driver, because the original is private
some times we were not sure, whether to duplicate code, or change the definition to be public